producer-c/producer-cloudwatch-integ/README.md

```diff
@@ -28,94 +28,97 @@ Create a build directory in the newly checked out repository, and execute CMake
 2. `cd amazon-kinesis-video-streams-demos/build`
 3. `cmake ..`
 
-The file installs the C Producer SDK libraries and PIC libraries for you.
+The file installs the C Producer SDK libraries, PIC libraries for you and CPP SDK components for you.
 
 NOTE: This project requires setting up of AWS SDK CPP Libraries. The specific components being used are:
 1. `events`
 2. `monitoring`
 3. `logs`
 
-You can refer to the build steps here: `https://github.com/aws/aws-sdk-cpp` . The synopsis of the build steps utlized to get this project working is summarized below:
-
-1. Create a directory that will store all the AWS SDK required libraries:
-`mkdir aws-libs`
-
-2. Install AWS Common Library
-```
-git clone https://github.com/awslabs/aws-c-common
-cd aws-c-common
-mkdir build && cd build
-cmake .. -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=<path/to/aws-libs> 
-make 
-sudo make install
-cmake .. -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX=<path/to/aws-libs> 
-make 
-sudo make install
-```
-
-3. Install AWS Checksums Library
-```
-git clone https://github.com/awslabs/aws-checksums
-cd aws-checksums
-mkdir build && cd build
-cmake .. -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=<path/to/aws-libs> 
-make 
-sudo make install
-cmake .. -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX=<path/to/aws-libs> 
-make 
-sudo make install
-```
-
-4. Install AWS Event Streams Library
-```
-git clone https://github.com/awslabs/aws-c-event-stream
-cd aws-c-event-stream
-mkdir build && cd build
-cmake .. -DBUILD_SHARED_LIBS=ON  -DCMAKE_MODULE_PATH=<path/to/aws-libs>/lib/cmake -DCMAKE_INSTALL_PREFIX=<path/to/aws-libs> 
-make 
-sudo make install
-cmake .. -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX=<path/to/aws-libs> 
-make 
-sudo make install
-```
-
-5. Finally, install the AWS CPP SDK library. This step takes some time. 
-```
-git clone https://github.com/aws/aws-sdk-cpp.git
-cd aws-sdk-cpp
-mkdir build && cd build
-export LD_LIBRARY_PATH=<path/to/aws-libs>/lib/
-cmake .. -DBUILD_ONLY="monitoring;events;logs" -DBUILD_DEPS=OFF -DCMAKE_MODULE_PATH=<path/to/aws-libs>/lib/cmake/ -DCMAKE_PREFIX_PATH="<path/to/aws-libs>/lib/aws-c-event-stream/cmake/;<path/to/aws-libs>/lib/aws-c-common/cmake/;<path/to/aws-libs>/lib/aws-checksums/cmake/;<path/to/aws-libs>/lib/cmake/"
-make 
-sudo make install
-```
-
-The cloudwatch library will now be ready to use!
-
-
 ## Running the application
 
 The demo comprises of a simple sample that uses custom constructed frames to capture certain metrics and performance parameters of the C Producer SDK and PIC. To run the sample:
 
-`./kvsProducerSampleCloudwatch <path-to-config-file>`
+`./kvsProducerSampleCloudwatch <path-to-config-file>`, or
+`./kvsProducerSampleCloudwatch <path-to-config-file>`	
```

Comment:
Did you mean to modify the command line after the or

Reply:
Oops! Fixed